GitUp/Application/Info.plist

```diff
@@ -33,6 +33,19 @@
 	<string>${BUNDLE_VERSION}</string>
 	<key>CFBundleShortVersionString</key>
 	<string>${BUNDLE_VERSION_STRING}</string>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLName</key>
```

Comment:
The `CFBundleURLName` key is not present in the GitHub Desktop Mac app, they only have this:
```
	<key>CFBundleURLTypes</key>
	<array>
		<dict>
			<key>CFBundleURLName</key>
			<string>com.github.GitHub</string>
			<key>CFBundleURLSchemes</key>
			<array>
				<string>github-mac</string>
			</array>
		</dict>
	</array>
```

According to https://developer.apple.com/library/ios/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html#//apple_ref/doc/uid/TP40009249-SW1 this key is not required, so let's remove it to match GitHub.